src/components/NewPosition/DepositSelector/DepositSelector.tsx

```diff
@@ -32,6 +32,9 @@ export interface IDepositSelector {
   isCurrentPoolExisting: boolean
   className?: string
   progress: ProgressState
+  balanceValue: string
+  percentageChange: number
```

Comment:
these two props might be just optional, also they should be doubled, one set for each token